nncf/openvino/rt_info.py

```diff
@@ -17,6 +17,23 @@
 from nncf.scopes import IgnoredScope
 
 
+def handle_ignored_scope(value: IgnoredScope) -> IgnoredScope:
```

Comment:
We can return here `Dict[str, Any]` instead of `IgnoredScope`, because we use `__dict__` after that

```suggestion
def handle_ignored_scope(value: IgnoredScope) -> Dict[str, Any]:
```